source/native/symbulation_parasite.cc

```diff
@@ -28,6 +28,20 @@ int symbulation_main(int argc, char *argv[]) {
   SymConfigSGP config;
   CheckConfigFile(config, argc, argv);
 
+  // stress hard-coded transmission modes
+  if (config.ORGANISM_TYPE() == 2) {
```

Comment:
We can create an organism type enumerated class to check against instead of directly checking against the numbers (to improve readability).

Reply:
(same for the checks below)